samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -2447,29 +2340,28 @@ mfxStatus CTranscodingPipeline::InitEncMfxParams(sInputParams *pInParams)
 #if (MFX_VERSION >= 1025)
     if(pInParams->numMFEFrames || pInParams->MFMode)
     {
-        m_ExtMFEParam.MaxNumFrames = pInParams->numMFEFrames;
-        m_ExtMFEParam.MFMode = pInParams->MFMode;
-
-        m_EncExtParams.push_back((mfxExtBuffer*)&m_ExtMFEParam);
+        auto mfePar = m_mfxEncParams.AddExtBuffer<mfxExtMultiFrameParam>();
+        mfePar->MaxNumFrames = pInParams->numMFEFrames;
+        mfePar->MFMode = pInParams->MFMode;
     }
 
     if(pInParams->mfeTimeout)
     {
-        m_ExtMFEControl.Timeout = pInParams->mfeTimeout;
-        m_EncExtParams.push_back((mfxExtBuffer*)&m_ExtMFEControl);
+        auto mfeCtrl = m_mfxEncParams.AddExtBuffer<mfxExtMultiFrameControl>();
+        mfeCtrl->Timeout = pInParams->mfeTimeout;
     }
 #endif
     if (m_pParentPipeline && m_pParentPipeline->m_pmfxPreENC.get())
     {
-        m_mfxEncParams.mfx.RateControlMethod       = MFX_RATECONTROL_LA_EXT;
-        m_mfxEncParams.mfx.EncodedOrder            = 1; // binary flag, 0 signals encoder to take frames in display order
-        m_mfxEncParams.AsyncDepth = m_mfxEncParams.AsyncDepth == 0 ? 2: m_mfxEncParams.AsyncDepth;
+        m_mfxEncParams.mfx.RateControlMethod = MFX_RATECONTROL_LA_EXT;
```

Comment:
looks like you have occasionaly changed formatting in code below?

Reply:
on purpose